### ISSUE TITLE
Stride.Launcher - Could not find a part of the path

### ISSUE THREAD

**Issue by VaclavElias** [stride3d/stride#1185]
**Release Type**: GitHub

**Version**: master [a265571](https://github.com/stride3d/stride/commit/a265571f34e859f2037e1d73eeae5ea35d8bfd37)

**Platform(s)**: Windows

**Describe the bug**
When I launch Stride.Launcher from the code I get this error:
System.IO.DirectoryNotFoundException: 'Could not find a part of the path 'C:\Users\Vacla\.nuget\packages\stride.gamestudio\4.1.0.1459-beta\tools'.'

I can ignore it and the Luancher is working normally.

**To Reproduce**
1. Clone master branch
2. Open Stride.Launcher.sln in VS 2022
3. Rebuild
4. Run Stride.Launcher

The same issue happened also when I used VS 2019 a few days ago..

**Expected behavior**
No error.

**Screenshots**
![image](https://user-images.githubusercontent.com/4528464/141685704-fa304da5-095c-4119-ae1f-a1d67b87ce03.png)

The location doesn't contain tools

![image](https://user-images.githubusercontent.com/4528464/141685894-360bf088-2ceb-43b1-a6e9-1021cb903ca6.png)

**Comment by VaclavElias** [comment]
This error message doesn't show up with master-4.0 branch.

**Comment by jrinker03** [comment]
I found the exception still happens in the master branch but is swallowed.  

[Per the fix](https://github.com/stride3d/stride/commit/de0a0cb6dd9e6726ee4945cd6296ee34480c0c98) for #1139, the executables were moved to the tools directory on September 1, 2021 but the current 4.1 package, 4.1.0.1459-beta, was uploaded on May 10, 2021 so does not have a tools directory.

I'd suggest a minimal change would be to make the code @VaclavElias shows in the first post more robust to path changes by checking for the existence of each target directory before trying to delve into it.

I'm happy to make that change if this makes sense.
